docker-compose.yml

```diff
@@ -79,16 +79,16 @@ services:
 
   streaming:
     build: .
-    image: ghcr.io/mastodon/mastodon:v4.2.11
+    image: ghcr.io/mastodon/mastodon-streaming:v4.3.0-beta.1
     restart: always
     env_file: .env.production
-    command: node ./streaming
+    command: node ./streaming/index.js
     networks:
       - external_network
       - internal_network
     healthcheck:
       # prettier-ignore
-      test: ['CMD-SHELL', 'wget -q --spider --proxy=off localhost:4000/api/v1/streaming/health || exit 1']
+      test: ['CMD-SHELL', "curl -s --noproxy localhost localhost:4000/api/v1/streaming/health | grep -q 'OK' || exit 1'"]
```

Comment:
There seems to be an erroneous `'` in this healthcheck after the `exit 1`

Reply:
Yes it has been fixed by #31612 for the next beta.